resources/views/community/components/user/profile-meta.blade.php

```diff
@@ -0,0 +1,68 @@
+@props([
+    'averageCompletionPercentage' => '0.0',
+    'averageFinishedGames' => 0,
+    'averagePointsPerWeek' => 0,
+    'developerStats' => [],
+    'hardcoreRankMeta' => [],
+    'recentPointsEarned' => [],
+    'socialStats' => [],
+    'softcoreRankMeta' => [],
+    'totalHardcoreAchievements' => 0,
+    'totalSoftcoreAchievements' => 0,
+    'userClaims' => null, // ?array
+    'userMassData' => [],
+    'username' => '',
+])
+
+<?php
+use App\Site\Enums\Permissions;
+
+$jrDevPermission = Permissions::JuniorDeveloper;
+?>
+
+<div class="relative">
+    <x-user.profile.primary-meta
+        :hardcoreRankMeta="$hardcoreRankMeta"
+        :softcoreRankMeta="$softcoreRankMeta"
+        :userMassData="$userMassData"
+        :username="$username"
+    />
+
+    <div class="mb-2 sm:ml-[142px] md:ml-0 md:absolute md:top-0.5 md:right-0 lg:relative lg:ml-[142px] xl:absolute xl:ml-0">
+        <x-user.profile.social-interactivity :username="$username" />
+    </div>
```

Comment:
This widget should be hidden for users with permissions <= 0. (Improvement over past behavior).

Reply:
It may be desirable to add a Banned/Spam indicator instead as that information is no longer visible in the title card.

Before:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/182b8cbc-c005-4963-8340-5b1bdc2da16c)

Reply:
Good idea, adjusted in latest. The legacy role indicator now also appears for all legacy roles that are not `Registered` or `Unregistered`.